src/Ledger/Ratify.lagda

```diff
@@ -351,42 +349,45 @@ as well as \isCC, \isDRep, and \isSPO, which return \true if the given delegate
 {\small
 \begin{code}
 getStakeDist : GovRole → ℙ VDeleg → StakeDistrs → VDeleg ⇀ Coin
-getStakeDist CC    cc  _                             = constMap (filterˢ isCCProp cc) 1
-getStakeDist DRep  _   record { stakeDistr = dist }  = filterᵐ (sp-∘ isDRepProp proj₁) dist
-getStakeDist SPO   _   record { stakeDistr = dist }  = filterᵐ (sp-∘ isSPOProp proj₁) dist
+getStakeDist role cc record { stakeDistr = dist } = case role of λ where
+  CC    → constMap (filterˢ isCCProp cc) 1
+  DRep  → filterᵐ (sp-∘ isDRepProp  proj₁) dist
+  SPO   → filterᵐ (sp-∘ isSPOProp   proj₁) dist
 
 acceptedStake : GovRole → ℙ VDeleg → StakeDistrs → (VDeleg ⇀ Vote) → Coin
 acceptedStake r cc dists votes =
   Σᵐᵛ[ x ← (getStakeDist r cc dists ∣ votedYesHashes votes r) ᶠᵐ ] x
 
 totalStake : GovRole → ℙ VDeleg → StakeDistrs → (VDeleg ⇀ Vote) → Coin
-totalStake r cc dists votes = Σᵐᵛ[ x ← getStakeDist r cc dists ∣ votedAbstainHashes votes r ᶜ ᶠᵐ ] x
+totalStake r cc dists votes = Σᵐᵛ[ x  ← getStakeDist r cc dists
+                                      ∣ votedAbstainHashes votes r ᶜ ᶠᵐ ] x
 
 activeVotingStake : ℙ VDeleg → StakeDistrs → (VDeleg ⇀ Vote) → Coin
-activeVotingStake cc dists votes = Σᵐᵛ[ x ← getStakeDist DRep cc dists ∣ dom (votes ˢ) ᶜ ᶠᵐ ] x
+activeVotingStake cc dists votes = Σᵐᵛ[ x  ← getStakeDist DRep cc dists
+                                           ∣ dom (votes ˢ) ᶜ ᶠᵐ ] x
 
 -- For now, consider a proposal as accepted if the CC and half of the SPOs
 -- and DReps agree.
 accepted' : RatifyEnv → EnactState → GovActionState → Set
-accepted' Γ es@record { cc = cc , _    ; pparams = pparams , _ }
-            s@record  { votes = votes' ; action = action } =
-  acceptedBy CC ∧ acceptedBy DRep ∧ acceptedBy SPO ∧ meetsMinAVS
-  where
-    open RatifyEnv Γ
-    open PParams pparams
+accepted' Γ (record { cc = cc , _ ; pparams = pparams , _ }) gs =
+  let
```

Comment:
I again prefer a `where` here. It makes the condition to be accepted immediately clear instead of having to look way below.